tests/drivers/spi/spi_loopback/boards/mr_canhubk3.conf

```diff
@@ -0,0 +1,4 @@
+# Copyright 2023 NXP
+# SPDX-License-Identifier: Apache-2.0
+
+CONFIG_SPI_MCUX_LPSPI_DMA=n
```

Comment:
Is this overlay required? I would expect `CONFIG_SPI_MCUX_LPSPI_DMA` to default to N for this test

Reply:
right, is not necessary, deleted